java/java-hello-world/.vscode/launch.json

```diff
@@ -9,9 +9,11 @@
         "type": "cloudcode.kubernetes",
         "request": "launch",
         "skaffoldConfig": "${workspaceFolder}/skaffold.yaml",
+        "profile": "dockerfile",
         "watch": true,
         "cleanUp": true,
-        "portForward": true
+        "portForward": true,
+        "imageRegistry": "gcr.io/cloud-code-vs-code"
```

Comment:
I don't think this should be filled in (users can specify their own imageRegistry i think)